brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1065 Improvement: Admin - Add core category on blocks page.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1065

### Screenshots

- https://share.getcloudapp.com/7Ku6LYo5

### Types of changes




- Improvement

### How has this been tested?

- Go to Admin > Block page.
- Check the filter list. Click on the Core filter.
- When clicking on Core it should display the container, heading, buttons, and image blocks inside it.

### Checklist:
